game/templates/game/landing.html

```
@@ -447,6 +447,79 @@ <h4>Legal</h4>
+  
+          <!-- Footer: I Agree button -->
+          <div style="padding:16px 28px; border-top:1px solid rgba(255,255,255,0.07); display:flex; justify-content:flex-end; flex-shrink:0; background:#131522;">
+              <button onclick="document.getElementById('termsModal').style.display='none'" style="background:linear-gradient(135deg,#f0c040,#d4a020); color:#0f0f1a; border:none; padding:10px 28px; border-radius:8px; font-weight:700; font-size:0.95rem; cursor:pointer; transition:all 0.2s ease; box-shadow:0 4px 16px rgba(240,192,64,0.3);">I Agree</button>
+          </div>
+  
+      </div>
+  </div>
+
     <div id="backToTopWrapper">
       <span id="backToTopTooltip">Back to Top</span>
       <button id="backToTopBtn" aria-label="Back to top" onclick="window.scrollTo({top:0,behavior:'smooth'})">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)